cpu/gd32v/Makefile.features

```diff
@@ -1,3 +1,5 @@
+CPU_CORE := rv32imac
```

Comment:
Probably the Kconfig should also be modified.
https://github.com/RIOT-OS/RIOT/blob/master/cpu/riscv_common/Kconfig
https://github.com/RIOT-OS/RIOT/blob/master/cpu/gd32v/Kconfig

Reply:
you might want to have another look at the next two commits